code/obj/item/assembly/misc_assemblies.dm

```diff
@@ -85,6 +85,7 @@ Contains:
 		checked_item.master = src
 		checked_item.layer = initial(checked_item.layer)
 		src.w_class = max(src.w_class, checked_item.w_class)
+		src.contraband = max(src.contraband, checked_item.contraband)
```

Comment:
This doesn't actually work, see `/datum/component/contraband` (I think there's still some spots where this is getting set directly elsewhere but they are Wrong and I tested it)